src/view/app/Indexes/index.tsx

```diff
@@ -1,21 +1,6 @@
 import { createRoot } from 'react-dom/client';
-import { TableDetails } from '@app/model';
 import './indexes.css';
 import Indexes from './indexes';
-import { ISettings } from '@src/common/IExtensionSettings';
-
-declare global {
-    interface Window {
-        acquireVsCodeApi(): any;

```

Comment:
Since the `acquireVsCodeApi` and `configuration` are available:
![image](https://github.com/BalticAmadeus/ProBro/assets/44312404/5365f3c1-d4ca-48f6-812d-f27e4f15ab93)

I would argue to leave these declarations here for now.